tools/duplicate_function_eliminator.py

```diff
@@ -247,8 +247,11 @@ def write_chunk(chunk, i):
       if DEBUG: print >> sys.stderr, 'splitting up js optimization into %d chunks, using %d cores  (total: %.2f MB)' % (len(chunks), cores, total_size/(1024*1024.))
       pool = multiprocessing.Pool(processes=cores)
       filenames = pool.map(run_on_chunk, commands, chunksize=1)
-      pool.terminate()
-      pool.join()
+      try:
+        pool.terminate()
+        pool.join()
+      except Exception, e:
```

Comment:
I still don't get it. What is being caught - why is an error now possible that was not before?